hack/deployer/config/plans.yml

```diff
@@ -58,7 +58,7 @@ plans:
   operation: create
   clusterName: dev
   provider: aks
-  kubernetesVersion: 1.24.6
+  kubernetesVersion: 1.24.10
```

Comment:
Why not the latest version?

Reply:
It doesn't seem that 1.24.11 or .12 is available from the cli:

```
❯ az aks get-versions -l westeurope | jq '.orchestrators[].orchestratorVersion'
"1.23.12"
"1.23.15"
"1.24.9"
"1.24.10"
"1.25.5"
"1.25.6"
"1.26.0"
```

And the latest release only notes 1.24.10
https://github.com/Azure/AKS/releases/tag/2023-03-26

Or are you suggesting to move to 1.26? If so I'm testing 1.26 locally now to ensure functionality prior to trying to move up 2 major versions